TYPO3.Media/Classes/TYPO3/Media/Domain/Service/ImageService.php

```diff
@@ -143,8 +144,23 @@ public function processImage(FlowResource $originalResource, array $adjustments)
             $imagineImage = $this->applyAdjustments($imagineImage, $adjustments, $adjustmentsApplied);
         }
 
+        $qualityAdjustments = array_filter($adjustments, function ($v) {
+            return $v instanceof QualityImageAdjustment;
+        }, ARRAY_FILTER_USE_BOTH);
+        if (count($qualityAdjustments) > 0) {
+            /** @var QualityImageAdjustment $qualityAdjustment */
+            $qualityAdjustment = array_pop($qualityAdjustments);
+            $quality = $qualityAdjustment->getQuality();
+            if ($quality >= 1 && $quality <= 100) {
+                $additionalOptions['quality'] = $qualityAdjustment->getQuality();
+                $adjustmentsApplied = true;
+            }
```

Comment:
Other Adjustment Models verify with their method canBeApplied if the given value make sense or not and are just not applied, so no exceptions here. Therefore it would be consistent with the behavior of the other adjustments, even if Quality is not really like an adjustment as it's a parameter applied to the save method of the image.